gradle/libs.versions.toml

```diff
@@ -9,6 +9,7 @@
 # Baselines, should be updated on every release
 baseline-core-api = "3.6.10"
 baselinePerfCore = "3.6.10"
+baselinePerfCoreMicrometer = "1.1.9"
```

Comment:
Is there any reason why you chose 1.1.9 instead of 1.1.10?

Reply:
Hah, I took the most recent version from my local Maven repo; should have checked Maven Central instead, of course.